lib/monkey/mk_core/mk_event_epoll.c

```diff
@@ -138,6 +138,9 @@ static inline int _mk_event_add(struct mk_event_ctx *ctx, int fd,
     }
 
     event->mask = events;
+    event->priority = MK_EVENT_PRIORITY_DEFAULT;
```

Comment:
Unfortunately, I could only test epoll. Need to discuss testing the other backends.

Reply:
Also tested LibEvent on windows.

Now only KQueue and Select remain.